Dockerfile

```diff
@@ -62,6 +62,13 @@ RUN set -x && \
     apt-get install -y nodejs && \
     npm install -g npm@6
 
+# Add yarn
+RUN set -x && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
```

Comment:
The two lines above could be moved to the top (before apt-get is called for the first time) so we have full setup (i.e. repos and keys) before the first `apt-get update`

Reply:
I've tried that, base image doesn't have enough tools to run `apt-key add` command:
```
E: gnupg, gnupg2 and gnupg1 do not seem to be installed, but one of them is required for this operation
```
So I'm afraid that we have to go with double `apt-get update`